src/ess/dream/workflow.py

```diff
@@ -85,7 +85,7 @@ def default_parameters() -> dict:
 def _collect_reducer_software() -> ReducerSoftwares:
     return ReducerSoftwares(
         [
-            Software.from_package_metadata('essdiffraction'),
+            # Software.from_package_metadata('essdiffraction'),
```

Comment:
Why?

Reply:
Left over from local development.
It's an issue for me because of the way I locally 'installed' our packages (using symbolic links instead of a `pip install`).
I'll change it back.